src/transformers/utils/quantization_config.py

```diff
@@ -672,6 +681,43 @@ def post_init(self):
                     f"Required fields are missing in the fusing mapping, required fields are {required_keys}"
                 )
 
+        if self.use_exllama is None:
+            # Default behaviour
+            self.use_exllama = False
+
+        if self.exllama_config is None:
+            self.exllama_config = {"version": ExllamaVersion.ONE}
```

Comment:
Is there a reason why we don't default to the exllamav2 ? In GPTQ, I couldn't change it since it will not be backward compatible.

Reply:
Nope just following GPTQ integration, I'll set it to v2

Reply:
done!